untrusted/quote_verify/src/lib.rs

```diff
@@ -185,14 +244,19 @@ impl Quote {
 pub enum Error {
     /// Unable to load the Certificate with mbedtls
     Certificate(mbedtls::Error),
+
     /// Failure to parse the Pem files from the quote data
     PemParsing(PemError),
+
     /// Failure to verify a Signature
     Signature(mbedtls::Error),
+
     /// A failure to convert a binary key into an mbedtls version.
     /// This is unlikely to happen as it should only happen if there is an
     /// error in the FFI or if there is a failure to malloc.
     Key(mbedtls::Error),
+
+    AttestationKey,
```

Comment:
Let's add rustdoc for this
```suggestion
    /// Invalid attestation key in quote.
    AttestationKey,
```

Reply:
done